content/solutions/pipeline.html.haml

```diff
@@ -0,0 +1,142 @@
+        plugin](https://wiki.jenkins-ci.org/display/JENKINS/Workflow+Plugin),
+        users now can implement a project's entire build/test/deploy pipeline
+        in a `Jenkinsfile` and store that alongside their code, treating their
+        pipeline as another piece of code checked into source control.
+
+        The Workflow plugin itself is inspired by the [Build Flow
+        plugin](https://wiki.jenkins-ci.org/display/JENKINS/Build+Flow+Plugin)
+        but aims to improve upon some concepts explored by Build Flow with
+        features like the ability to suspend/resume of executing jobs.
+
+        In an adjacent space is the [Job DSL
+        plugin](https://wiki.jenkins-ci.org/display/JENKINS/Job+DSL+Plugin)
+        which is worth checking out depending on your use-case as well.
+
+
+        ## Getting Started
+
+        * The [Workflow
+          plugin](https://wiki.jenkins-ci.org/display/JENKINS/Workflow+Plugin)
+          has a fairly comprehensive
+          [tutorial](https://github.com/jenkinsci/workflow-plugin/blob/master/TUTORIAL.md)
+          checked into its source tree.
+        * [Overview of Continuous Delivery with Workflow](http://documentation.cloudbees.com/docs/cookbook/_continuous_delivery_with_jenkins_workflow.html)
+        * [DZone Refcard](https://dzone.com/refcardz/continuous-delivery-with-jenkins-workflow)
+        * [Pipeline as code "cook book"](http://documentation.cloudbees.com/docs/cookbook/pipeline-as-code.html)
+
+        ## Additional resources
+
+        * [Workflow Plugin page](https://wiki.jenkins-ci.org/display/JENKINS/Workflow+Plugin)
+        * [Plugins compatible with Workflow](https://github.com/jenkinsci/workflow-plugin/blob/master/COMPATIBILITY.md)
+        * [Stack Overflow "jenkins-workflow" tag](http://stackoverflow.com/questions/tagged/jenkins-workflow)
+        * [Examples repository](/content/workflow-best-practices-and-examples-repo-github)
+
+
+        ## Presentations
+
+        The presentations below were given by [Jesse Glick](https://github.com/jglick), author of the [Workflow Plugin page](https://wiki.jenkins-ci.org/display/JENKINS/Workflow+Plugin)
+
+        <iframe width="560" height="315" src="https://www.youtube-nocookie.com/embed/VkIzoU7zYzE" frameborder="0" allowfullscreen></iframe>
+
+        * [2014 presentation](https://www.youtube.com/watch?v=gpaV6x9QwDo) (with [slides](https://www.cloudbees.com/sites/default/files/2014-0618-Boston-Jesse_Glick-Workflow.pdf))
+
+        Other presentations:
+
+        * [Pimp your Continuous Delivery Pipeline with Jenkins workflow (W-JAX 14)](http://www.slideshare.net/cloudbees/pimp-your-continuous-delivery-pipeline-with-jenkins-workflow-wjax-14) by [Cyrille Le Clerc](https://github.com/cyrille-leclerc)
+
+      %div#content-bottom/
+
+- else
+  - # New and improved version of the Pipeline solution page
+  = partial('toptoolbar.html.haml',
+    :section=>'more',
+    :specific=>'docker')
```

Comment:
Super minor, as this is for the non-legacy site, but `docker` -> `pipeline`?  Or will that break the build due to that type of toolbar being missing?